src/components/Treeview/Label.js

```diff
@@ -21,15 +21,23 @@ export const Indicator = styled(Icon).attrs({
 	)};
 `;
 
+export const BeforeIndicator = styled.div`
+	background-color: #666;
+	margin-left: -${props => branchLength(props) + 1}px;
+	width: ${props => branchLength(props) + 1}px;
+	height: 1px;
+	z-index: 99;
+	position: absolute;
+`;
```

Comment:
I'd like an explanation of what this element does, and why it is necessary. As far as I can tell it's a horizontal line, so overlaps in function with NonIndicator below it. They're also used together. Why can't it be a single element?

Reply:
I checked the code out and went over it, this seems fine, overall.